lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -167,7 +170,14 @@ import qualified Data.Text.Encoding as T
 --
 -- @m | purpose' | cointype' | account' | role | address@
 data Depth
-    = RootK | PurposeK | CoinTypeK | AccountK | RoleK | AddressK | ScriptK
+    = RootK
+    | PurposeK
+    | CoinTypeK
+    | AccountK
+    | RoleK
+    | AddressK
+    | ScriptK
+    | PolicyK
```

Comment:
Better update all the comments about `Depth`, starting with the one just above.

Reply:
Mmmm... not ideal. Kinda brutalizing the abstraction here, but 'ScriptK did it first and it's the path of least resistance...

Reply:
Updated doco

Reply:
Yeah if it weren't for ScriptK I would have asked you to try and find a better way of representing depth.